data-prepper-plugins/rds-source/src/main/java/org/opensearch/dataprepper/plugins/source/rds/model/ExportStatus.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.source.rds.model;
+
+import java.util.Set;
+
+public enum ExportStatus {
+    CANCELED,
+    CANCELING,
+    COMPLETE,
+    FAILED,
+    IN_PROGRESS,
+    STARTING;
+
+    public static final Set<String> TERMINAL_STATUS_NAMES = Set.of(CANCELED.name(), COMPLETE.name(), FAILED.name());
```

Comment:
I'd recommend `isTerminal()` as a method on this. Then, you can make this private and move the `contains()` call into here. But, I'm fine either way.

Reply:
I see what you mean here. Added the method and unit tests.